python-libraries/data-platform-catalogue/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.0] 2024-02-14
+
+### Changed
+
+- Added subdomain property to dataproduct
```

Comment:
Might want to note removal of openmetadata too

Reply:
Included